src/models/Movie/type.ts

```diff
@@ -0,0 +1,22 @@
+import { Entry, EntrySkeletonType, FieldsType } from 'contentful';
+
+export type MovieModel = {
```

Comment:
nit: I'd name this just `Movie` for 2 reasons:
- keep consistent with the existing `Genre`
- this is just a type and not really a model, though I'm not sure if for this application we're actually going to implement models